src/main/java/it/infuse/jenkins/usemango/UseMangoTestExecutor.java

```diff
@@ -92,20 +96,38 @@ public synchronized void run() {
 		            	test.setPassed(false);
 		            	listener.getLogger().println("FAIL: Test '"+test.getName()+"' failed");
 		            }
-					
-					// ALLUSERPROFILE returns C:\ProgramData
-					FilePath junitPath = new FilePath(currentNode.getChannel(), currentNode.toComputer().getEnvironment().get("ALLUSERSPROFILE"));
-
-					// write result to workspace (junit)
-					String junit = IOUtils.toString(junitPath
-							.child("\\useMango\\Logs\\junit.xml").read(), StandardCharsets.UTF_8.name());
-					
-					workspace.child(ProjectUtils.RESULTS_DIR).
-						child(ProjectUtils.getJUnitFileName(test.getId())).write(junit, StandardCharsets.UTF_8.name());
-					
-					listener.getLogger().println("STOP: Outcome saved to workspace for test '"+test.getName()+"'");
-					
-				} catch (IOException | InterruptedException e) {
+
+                    String umLogPath = "\\useMango\\Logs\\junit.xml";
+                    String localLogPath;
+
+                    String workingDataPath = WinRegistry.readString(WinRegistry.HKEY_LOCAL_MACHINE, "SOFTWARE\\WOW6432Node\\Infuse Consulting\\Mango", "WorkingDataPath");
```

Comment:
What happens if this registry value isn't found? It won't be there on existing versions of useMango, so should fall back to the location used by the v4.4 application.